recipes/vs2008_express_vc_python_patch/meta.yaml

```diff
@@ -0,0 +1,24 @@
+package:
+  name: vs2008_express_vc_python_patch
+  version: 1.0
+
+source:
+  url: https://github.com/conda-forge/conda-forge-build-setup-feedstock/raw/master/vs2008_patch.zip
```

Comment:
We need this file.

xref: https://github.com/menpo/condaci/blob/98d76b2089d494433ac2e3abd920123088a95a6d/vs2008_patch.zip